HW1/P3/P3.txt

```diff
@@ -1 +1,6 @@
-# Your discussion here
+Assuming that we want the first one in alphabetical order, the single entry with the most anagrams is: 
+
+************ New Letter Sequence ****************
+Sorted Letter Sequence: aelst
+Number of Valid Anagrams: 11
+Words: least, leats, salet, slate, stale, steal, stela, taels, tales, teals, tesla, 
```

Comment:
Yeah, this may be different than what other people got because I sorted my RDD byKey. So I got mine out in alphabetical order. But to add to your point, I think there were over 4 or maybe 5 lists of 11.